lib/couchdb/index.js

```diff
@@ -84,23 +85,59 @@ const upsert = R.curry((couch, bucket, keys, doc) =>
 )(doc)))
 
 
+const _findOneByIdWhereEq = R.curry((couch, bucket, id, predicates) =>
+  getById(couch, bucket, id)
+  .then(R.when(
```

Comment:
Yes they should be the same. I didn't remember R.whereEq existed when I did this.